lib/matplotlib/tests/test_transforms.py

```diff
@@ -381,7 +379,6 @@ def test_line_extent_compound_coords1(self):
         assert_array_equal(ax.dataLim.get_points(),
                            np.array([[np.inf, -5.],
                                      [-np.inf, 35.]]))
-        plt.close()
```

Comment:
Already done by `teardown_method()`.

Reply:
Or even by the autouse()d _cleanup_cm? (i.e. you can even remove the whole teardown_method).